src/media-segment-request.js

```diff
@@ -165,11 +165,6 @@ const handleInitSegmentResponse =
 
   segment.map.bytes = new Uint8Array(request.response);
 
-  // Initialize CaptionParser if it hasn't been yet
```

Comment:
I think we were supposed to keep this one. Theoretically we can receive the init segment and first segment in any order, whichever is received first should init the caption parser.

Reply:
Although maybe this was in SL originally...

Reply:
Nevermind, I see this is now done in `handleSegmentBytes` which looks like the right way to do this